### ISSUE TITLE
Account; error message when saving

### ISSUE THREAD

**Issue by roswitina** [RichardDern/Cyca#22]
I use the Docker version.

I can create an account, confirm it and then login to the account. When I click Save in the account, I get the following message:

> Oops! An Error Occurred
> The server returned a "405 Method Not Allowed".
> 
> Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

Where can I have a look?

**Comment by RichardDern** [comment]
What's the URL in your browser when you get the 405 error ?

You can give me more informations by looking at your logs.

```
docker exec cyca_app_1 ls /app/storage/logs
```

Assuming _cyca_app_1_ is your container's name.

There you should have at least one file named after the current date. This file will contain details about what happened.

**Comment by roswitina** [comment]
I have Cyca in use on an intranet. The internal IP address is resolved over a domain DNS. This is http://cyca.intranet.lokal. 

I have now tried again and could not find the error message. The saving worked. But I did not find an error anymore. 

I have already set up an existing ReverseProxy (https://github.com/jc21/nginx-proxy-manager?utm_source=nginx-proxy-manager) with Let's Encrypt, which works very well for me. Can I use it without changing anything in the docker settings?

I have looked at the log files live with docker logs -f cyca_app_1 again but did not find any error message.
